admin_guide/install/advanced_install.adoc

```diff
@@ -83,22 +83,44 @@ the remaining instructions.
 
 == Configuring Ansible
 
-You must create an *_/etc/ansible/hosts_* file, which is Ansible's inventory
-file for the playbook to use during the installation.
+The *_/etc/ansible/hosts_* file is Ansible's inventory
+file for the playbook to use during the installation. Replace the contents of the file with your desired configuration.
 
-Any hosts you designate as masters during the installation process should also
-be configured as
+[NOTE]
+====
+Before running the Ansible installer, any hosts you intend to designate as
+masters during the installation process should also be configured as
 link:../../admin_guide/manage_nodes.html#marking-nodes-as-unschedulable-or-schedulable[unschedulable]
-nodes. This is so the masters are configured as part of the
+nodes, in order to configure the masters as part of the
 link:../../architecture/additional_concepts/networking.html#openshift-sdn[OpenShift
 SDN].
+====
 
-The following sections describe example inventory files for various environment
-topographies, including configuring
-link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#high-availability-masters[multiple
+The following sections describe example *_/etc/ansible/hosts_* files for various environment
+topographies, including example configurations for
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#high-availability-masters[using multiple
 masters for high availability]. You can choose an example that matches your
-requirements to modify and use as your inventory file when
-link:#running-the-ansible-installer[running the Ansible installer].
+requirements and use as your inventory file when
+link:#running-the-ansible-installer[running the Ansible installer], or modify an example to match your own environment.
+
+*Configuring Labels During the Ansible Inatall*
+
+To assign labels to hosts during the Ansible install, indicate the desired labels in the *_/etc/ansible/hosts_* file:
```

Comment:
Had that originally, but wasn't sure if could work for masters too. Mack to "node hosts" now.

And yes, done.